metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "saz-resolv_conf",
-  "version": "6.0.0",
+  "version": "7.0.0",
```

Comment:
Please don't increase the version of the module

Reply:
Sorry, I just wanted to stress out that it might be a breaking change.